DiffTests/PatchSortTests.swift

```diff
@@ -51,13 +51,12 @@ class PatchTests: XCTestCase {
             switch (element1, element2) {
             case (.insert(let at1), .insert(let at2)):
                 return at1 < at2
-            case (.insert, .delete):
+            case (.insert(_), .delete(_)):
```

Comment:
Ah, this is weird. You can check with swift people if it's a known issue (I bet it is).